snuba/web/rpc/v1/endpoint_trace_item_table.py

```diff
@@ -82,6 +91,9 @@ def _build_query(request: TraceItemTableRequest) -> Query:
             trace_item_filters_to_expression(request.filter),
         ),
         order_by=_convert_order_by(request.order_by),
+        groupby=[
+            attribute_key_to_expression(attr_key) for attr_key in request.group_by
```

Comment:
Will all the non-aggregations in the selected columns also be added to the group by? If not, this code should handle mismatches between selected columns and the groupby.

Reply:
yes good point